src/modules/storage.js

```diff
@@ -0,0 +1,34 @@
+export default class Store {
+  static getTasks = () => {
+    let tasks = [];
+
+    if (localStorage.getItem('tasks') === null) {
+      return tasks;
+    }
+
+    tasks = JSON.parse(localStorage.getItem('tasks'));
+    return tasks;
+  }
+
+  static addTask = (task) => {
+    const tasks = Store.getTasks();
+    tasks.push(task);
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  }
+
+  static removeTasks = (taskToDelete) => {
+    const tasks = Store.getTasks();
+    let newArr = [];
+    for (let i = 0; i < tasks.length; i += 1) {
+      if (tasks[i].completed === taskToDelete) {
+        newArr = tasks.filter((task) => task.completed === false);
+      }
+    }
+
+    for (let i = 0; i < newArr.length; i += 1) {
+      newArr[i].index = i;
+    }
```

Comment:
- [ ] The property index is set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5). So the index starts from 1, not from 0. Kindly implement it.